### DISCUSSION TITLE
Unicode lookup functionality?

### DISCUSSION THREAD

**Discussion by haakonstorm** [Flow-Launcher/Flow.Launcher#2438] [Plugin Request]
## **Is your feature request related to a problem? Please describe.**

Coming from macOS with Alfred, there are extensions to ease entering unicode and emoji characters.
E.g. 
`: smile`
Searches for emoji symbols with "smile" in its searchable description, providing a list of "smile" emojis to select from. I have installed the emoji picker extension for Flow Launcher which works great.
But for unicode, e.g. for finding foreign characters, rarely used symbols, and often used symbols like squared, cubed, etc, in Alfred the unicode extension (trigged with double colons), e.g:
`:: squared`
Searches for unicode symbols with "squared" in the description, so I can select the superscript 2 and insert it.

## **Describe the solution you'd like**

Type e.g.:
`:: squared`
And Flow Launcher searches through unicode characters, providing the superscript 2 to insert. 

## **Describe alternatives you've considered**

The Windows emoji picker (`WIN` key + `.` (dot)) works, but needs to be used with the mouse. Flow Launcher is for doing most things with the keyboard, right? 

## **Additional context**

The python source code for the Alfred unicode is here:
https://github.com/blueset/alfred-search-unicode

This project in turn depends on "uni" which is hosted here, written in Go and shell script:
https://github.com/arp242/uni

From what I can see, all that we need is to parse this:
https://raw.githubusercontent.com/arp242/uni/master/unidata/gen_codepoints.go

This is in a format like this:
```
	0x1001ef8: "Ytilde",
	0x1001ef9: "ytilde",
	0x1002070: "zerosuperior",
	0x1002074: "foursuperior",
	0x1002075: "fivesuperior",
	0x1002076: "sixsuperior",
	0x1002077: "sevensuperior",
	0x1002078: "eightsuperior",
	0x1002079: "ninesuperior",
	0x1002080: "zerosubscript",
	0x1002081: "onesubscript",
	0x1002082: "twosubscript",
	0x1002083: "threesubscript",
	0x1002084: "foursubscript",
	0x1002085: "fivesubscript",
	0x1002086: "sixsubscript",
	0x1002087: "sevensubscript",
	0x1002088: "eightsubscript",
	0x1002089: "ninesubscript",
	0x10020a0: "EcuSign",
	0x10020a1: "ColonSign",
	0x10020a2: "CruzeiroSign",
	0x10020a3: "FFrancSign",
	0x10020a4: "LiraSign",
	0x10020a5: "MillSign",
	0x10020a6: "NairaSign",
	0x10020a7: "PesetaSign",
	0x10020a8: "RupeeSign",
	0x10020a9: "WonSign",
```

I think this would be super useful for many people.
Thank you all for a great launcher and plug in ecosystem, making transitioning from macOS and Alfred to Windows a bit easier.

**Comment by jjw24** [comment]
Hi there, this appears to be a plugin request for searching unicode characters, converting to plugin request in discussions.

**Comment by z1nc0r3** [comment]
I just created a plugin for finding Unicord characters inspired by this discussion. Feel free to try it out: https://github.com/z1nc0r3/Unicode-Finder-Plugin. It'll take some time to populate on the plugin store btw.

**Comment by haakonstorm** [comment]
Hey, this is awesome and will be super useful! :-D

**Reply by NoPlagiarism** [reply]
Can you mark discussion as solved then?
